lib/mix/tasks/generate_alter_table_migrations.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Mix.Tasks.RailwayIpc.GenerateAlterTableMigrations do
+  use Mix.Task
```

Comment:
encoded_messages columsn need to be :text b/c encoded messages are not infrequently greater than 250 chars. This will generate a migration that apps using this package need to run when we ship the next release.